### PR TITLE
groovy: use branch name as deploy folder

### PR DESCRIPTION
**Pull request by jangernert**
Needs testing

### CODE REVIEW

**Comment by jangernert** [comment]
Tested successfully!
Deploys binaries to `Z:\releases\MetriCam2\git\$BRANCH_NAME\`.

**Comment by f00f** [comment]
Looks to me like it builds always the `master` branch:
```
def gitBranch = 'master'
// [...]
git branch: gitBranch, url: gitURL
```

File: Jenkinsfile.groovy [hunk 1]
```diff
@@ -54,7 +54,7 @@ pipeline {
 		}
 		stage('Deploy') {
 			environment {
-				def VERSION = 'latest'
+				def VERSION = getGitBranchName()
```
Comment (f00f) [inline Jenkinsfile.groovy]: `VERSION` is no longer an appropriate variable name

**Comment by jangernert** [comment]
retest this please

**Comment by jangernert** [comment]
test this please

**Comment by jangernert** [comment]
test this please

**Comment by jangernert** [comment]
test this please

**Comment by jangernert** [comment]
test this please

**Comment by jangernert** [comment]
test this please

**Comment by f00f** [comment]
Closed in favor of the cleaner #82

**Pull request closed without merging**